React_Day7/avengers/src/components/ErrorComponent.js

```diff
@@ -0,0 +1,17 @@
+import {useRouteError} from 'react-router-dom';
+
+const ErrorComponent = () => {
+
+    const err = useRouteError();
+
+  return (
+    <>
+        <h2>Sorry mate</h2>
+        <h2>What you're looking for isn't here.</h2>
+        <h4>Why? { err.message || err.statusText} </h4>
+        <a href='/'>Home</a>

```

Comment:
why anchor tag here.  can better you use Link here or Link in your header component better to always display it for all pages.